doc/lua_api.txt

```diff
@@ -2828,6 +2828,13 @@ Can be gotten via `item:get_meta()`.
 #### Methods
 * All methods in MetaDataRef
 
+### `StorageRef`
+Mod metadata: per mod metadata, saved automatically.
+Can be gotten via `minetest.get_mod_storage()`
```

Comment:
obtained is better than gotter, and add a precision about it's only at mod loading